.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*.{ps1,psm1}]
+indent_style = space
+indent_size = 4
+charset = utf-utf-8-bom
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Do we need this file?

Reply:
In general no, but I would still recommend having one as it would help people that have the editorconfig plugin installed in their preferred editor.
The plugin uses that file when running formatting on each file (I see now that I have a typo in that file though).